department-of-veterans-affairs/vets-api
# Update UserAcceptableVerifiedCredentialCountsJob to use StatsD

## Summary
Run Sidekiq cron job daily to set StatsD gauges with `user_avc` totals. ` user_avc_totals.{proivider}.{scope}.total`

- use StatsD gauge for user_avc Metrics
- Updated [docs](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Documentation/AVC_IVC_logs.md)

## How to test
- Start server - `bundle exec rails c`
- In different terminal start sidekiq - `bundle exec sidekiq`
- In different terminal tail statsd logs - `tail -f log/statsd.log`
- Go to `http://localhost:3000/sidekiq/recurring-jobs`
- Enqueue - `UserAcceptableVerifiedCredentialTotals`
- Check logs for ` user_avc_totals.*.*.total` statsd keys

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#5325


## Testing done
- Specs
- Ran locally, StatsD metrics show up
- Checked logs for errors 
- checked authentication 

## Screenshots


## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  Documentation has been updated ([link to documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Documentation/AVC_IVC_logs.md))

## Requested Feedback
